easybuild/easyconfigs/b/Bowtie/Bowtie-1.1.2-intel-2017a.eb

```diff
@@ -0,0 +1,23 @@
+# Modified from existing version by:
+# Robert Schmidt <rjeschmi@gmail.com>
+# Ottawa Hospital Research Institute - Bioinformatics Team
+name = 'Bowtie'
+version = '1.1.2'
+
+homepage = 'http://bowtie-bio.sourceforge.net/index.shtml'
+description = """Bowtie is an ultrafast, memory-efficient short read aligner.
+It aligns short DNA sequences (reads) to the human genome.
+"""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'pic': True}
+
+sources = ['%(namelower)s-%(version)s-src.zip']
+source_urls = ['http://download.sourceforge.net/bowtie-bio/']
+
+patches = [
+    'int64typedef.patch',
+    'void2int.patch'
```

Comment:
@masaofujinaga One last change request (which was also mentioned in one of the previous comments): please rename the patch file to `Bowtie-1.1.2_void2int.patch`, and use `Bowtie-%(version)s_void2int.patch` in the easyconfig file (to avoid hardcoding the Bowtie version in two places)

This is to make clear that the patch is most likely specific to this particular Bowtie version.